src/net.cpp

```diff
@@ -1968,25 +1972,43 @@ bool CConnman::InactivityCheck(const CNode& node) const
     if (!ShouldRunInactivityChecks(node, now)) return false;
 
     if (last_recv.count() == 0 || last_send.count() == 0) {
-        LogPrint(BCLog::NET, "socket no message in first %i seconds, %d %d peer=%d\n", count_seconds(m_peer_connect_timeout), last_recv.count() != 0, last_send.count() != 0, node.GetId());
+        LogDebug(BCLog::NET,
+            "socket no message in first %i seconds, %d %d disconnecting peer=%d%s\n",
+            count_seconds(m_peer_connect_timeout),
+            last_recv.count() != 0,
+            last_send.count() != 0,
+            node.GetId(),
+            fLogIPs ? strprintf(" peeraddr=%s", node.addr.ToStringAddrPort()) : ""
+        );
         return true;
     }
 
     if (now > last_send + TIMEOUT_INTERVAL) {
-        LogPrint(BCLog::NET, "socket sending timeout: %is peer=%d\n", count_seconds(now - last_send), node.GetId());
+        LogDebug(BCLog::NET,
+            "socket sending timeout: %is, disconnecting peer=%d%s\n", count_seconds(now - last_send),
+            node.GetId(),
+            fLogIPs ? strprintf(" peeraddr=%s", node.addr.ToStringAddrPort()) : ""
+        );
         return true;
     }
 
     if (now > last_recv + TIMEOUT_INTERVAL) {
-        LogPrint(BCLog::NET, "socket receive timeout: %is peer=%d\n", count_seconds(now - last_recv), node.GetId());
+        LogDebug(BCLog::NET,
+            "socket receive timeout: %is, disconnecting peer=%d%s\n", count_seconds(now - last_recv),
+            node.GetId(),
+            fLogIPs ? strprintf(" peeraddr=%s", node.addr.ToStringAddrPort()) : ""
+        );
         return true;
     }
 
     if (!node.fSuccessfullyConnected) {
+        const std::string disconnect_msg{strprintf(", disconnecting peer=%d%s",
```

Comment:
nit: Maybe this should move to the top of the function and be used in the other three disconnect logs here
```diff
 if (!ShouldRunInactivityChecks(node, now)) return false;
+const std::string disconnect_msg{...};
```